service/storage_service.hh

```diff
@@ -819,12 +824,30 @@ private:
 
     std::unordered_set<token> get_tokens_for(inet_address endpoint);
 
-    // Retrieves streams generation timestamp from the given endpoint's application state (broadcasted through gossip).
-    // We might be during a rolling upgrade, so the timestamp might not be there (if the other node didn't upgrade yet),
-    // but if the cluster already supports CDC, then every newly joining node will propose a new CDC generation,
-    // which means it will gossip the generation's timestamp.
+    /* Retrieves streams generation timestamp from the given endpoint's application state (broadcasted through gossip).
+     * We might be during a rolling upgrade, so the timestamp might not be there (if the other node didn't upgrade yet),
+     * but if the cluster already supports CDC, then every newly joining node will propose a new CDC generation,
+     * which means it will gossip the generation's timestamp.
+     */
```

Comment:
Why does this change belong to this patch?

Reply:
Fixd